### ISSUE TITLE
Encoded output is squash for some reason

### ISSUE THREAD

**Issue by mbojrab** [mbojrab/playbox#71]
The encoded vector is squash (not [0,1] space). Track this down and see if its affecting the cosine similarity results.

**Comment by mbojrab** [comment]
Found the encodings were scaled. This was due to using the softmax output not the logit. I updated this and verified the cosine similarity / training was always using the logit representation. I also verified in Orange that this was not causing any issues in clustering algorithms. Made update to logit output because its more intuitive.
